gammapy/estimators/points/profile.py

```diff
@@ -103,6 +109,14 @@ def __init__(self, regions, spectral_model=None, **kwargs):
             spectral_model = PowerLawSpectralModel()
 
         self.spectral_model = spectral_model
+
+        if "reoptimize" in kwargs:
+            if kwargs["reoptimize"] is True:
+                raise ValueError(
+                    f"reoptimize=True is not available for {self}. Set to `False` instead."
```

Comment:
I think we can  have a warning  "Reoptimisation not allowed for backward folding estimator. Setting reoptimize=False" and proceed by internally setting reoptimize=False always

Reply:
Using an error will then prevent to never be able to re-optimise. This is the wished behaviour?

Reply:
I think the answer is yes. 
Currently, it is not possible to re-optimise anyway, as you would get an error which is however much more unclear.